### ISSUE TITLE
Multiple themes cannot be imported

### ISSUE THREAD

**Issue by QFJABC** [t1m0thyj/WinDynamicDesktop#223]
- Windows version (10)
- WinDynamicDesktop version (4.2.0)
- Installation source (Installer)
- Language (Chinese)
Hello, I have an Issue with your program, Multiple themes cannot be imported! 
Every time I import more than five, it freezes at "Generating thumbnails, please wait..." for a long time. 
If I use the  "Task Manager" to end the process, the themes will be confused after restart.
The same problem occurs with the English version.
Please fix this problem, thank you!
![Bug 1](https://user-images.githubusercontent.com/49907719/86908863-caf44180-c149-11ea-9f57-76c78495cad8.jpg)
![Bug 2](https://user-images.githubusercontent.com/49907719/86908867-cd569b80-c149-11ea-9bb8-324802f536a4.jpg)

**Comment by QFJABC** [comment]
Hello, let me add more info about it. Because I downloaded Apple Wallpapers very slowly, so I downloaded them from this link https://github.com/t1m0thyj/WDD-mac-themes. At the same time, I downloaded some wallpapers from this website https://windd.info/themes/ (all .Dww files) and imported them one by one, this problem will occur randomly on any one (every time)

**Comment by gdstewart** [comment]
I've also noticed this error happening if you try to set a custom theme with faulty images. It will become stuck at "generating thumbnails" and if you end the process and restart the application, all of the themes including the default themes will be gone. The only solution I've found is to uninstall and reinstall the application. However, this is probably not a common situation (I only encountered it when bug testing the theme creator).
